modules/button/react/spec/__snapshots__/Button_Delete.snapshot.tsx.snap

```diff
@@ -15,23 +15,22 @@ exports[`Button (Delete) Snapshots renders a large, delete button 1`] = `
   -webkit-justify-content: center;
   -ms-flex-pack: center;
   justify-content: center;
-  font-size: 14px;
+  font-size: 13px;
```

Comment:
This isn't actually affecting anything because we set `fontSize: '14px'` on the nested span, But we should figure out why this is changing